k8s/namespaces/family-public-law/fpl-case-service/perftest.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fpl-case-service
+  annotations:
+    fluxcd.io/automated: "false"
+    hmcts.github.com/prod-automated: disabled
```

Comment:
do you know what this one does exactly? was not present in old approach

Reply:
after discussion with Praveen, these have been added to keep the perf-test version to a specific prod version number